### PR TITLE
Switch v1beta1 to v1

### PR DESCRIPTION
**Pull request by yaron2**
Closes https://github.com/dapr/dapr/issues/2354

### CODE REVIEW

**Review by youngbupark — APPROVED** [review]

**Comment by yaron2** [comment]
The CRD spec changed with k8s 1.16.. need to update the YAML too.

File: charts/dapr/crds/configuration.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```
Comment (youngbupark) [inline charts/dapr/crds/configuration.yaml:1]: Seems like CRD shema has been changed from v1beta to v1.

```
Error: failed to install CRD crds/components.yaml: CustomResourceDefinition.apiextensions.k8s.io "components.dapr.io" is invalid: [spec.versions: Invalid value: []apiextensions.CustomResourceDefinitionVersion(nil): must have exactly one version marked as storage version, status.storedVersions: Invalid value: []string(nil): must have at least one stored version]
Makefile:198: recipe for target 'docker-deploy-k8s' failed
```

Comment (youngbupark) [reply charts/dapr/crds/configuration.yaml:1]: Ah you fixed it 👍

Comment (youngbupark) [reply charts/dapr/crds/configuration.yaml:1]:
```
Error: failed to install CRD crds/components.yaml: CustomResourceDefinition.apiextensions.k8s.io "components.dapr.io" is invalid: spec.validation.openAPIV3Schema.type: Required value: must not be empty at the root
```

Generating CRD via controller-gen is much easier. See #2355

Comment (yaron2) [reply charts/dapr/crds/configuration.yaml:1]: Yeah using that.

**Comment by yaron2** [comment]
CRD schema is finally working, looks like the sidecar injector has an issue with `v1`. checking.

**Comment by mukundansundar** [comment]
I like the commit messages 👍 `black magic` `voodoo` 😄

**Review by youngbupark — APPROVED** [review]

**Pull request merged**